src/index.html

```diff
@@ -1,13 +1,44 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="img"
+        src="./images/imac.jpeg"
+        alt="#"
+      >
+
+      <div class="blocks">
+        <h3 class="blocks__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
+
+        <p class="phrase phrase--position_left">Код товара:195434</p>
+
+        <div  class = "stars">
+          <div class = "stars__star"></div>
+          <div class = "stars__star"></div>
+          <div class = "stars__star"></div>
+          <div class = "stars__star"></div>
+          <div class = "stars__star"></div>
+        Отзывов:5
+        </div>
+
+        <p class="price price--position_left">Цена:69 999грн</p>
+
+        <!-- <p class="price-price price-price--position_right">69 999грн</p> -->
+
+        <a
+          class="buy"
+          data-qa="hover"
+          href="#"> КУПИТЬ
+        </a>
```

Comment:
```suggestion
        <a
          class="buy"
          data-qa="hover"
          href="#"
        >
          купить  
        </a>
```
Зазвичай текст пишеться малими літерами, а в css при необхідності додається
text-transform: uppercase;